python/hail/methods/statgen.py

```diff
@@ -2167,11 +2164,12 @@ def balding_nichols_model(n_populations, n_samples, n_variants, n_partitions=Non
     fst : :obj:`list` of :obj:`float`, optional
         :math:`F_{ST}` values, a list of length ``n_populations`` with values
         in (0, 1). Default is ``[0.1, ..., 0.1]``.
-    af_dist : :class:`.UniformDist` or :class:`.BetaDist` or :class:`.TruncatedBetaDist`
+    af_dist : :class:`.Float64Expression` representing a random function.
```

Comment:
I think this should be a function taking a seed as an argument. I doubt users will read the documentation to find out that the randomness of this method depends on both the af_dist expression being the "same" and the seed parameter being the same.